ares/ms/cartridge/cartridge.cpp

```diff
@@ -23,8 +23,15 @@ auto Cartridge::connect() -> void {
   if(information.board == "Codemasters") board = new Board::Codemasters{*this};
   if(information.board == "Korea") board = new Board::Korea{*this};
   if(information.board == "MSX") board = new Board::MSX{*this, 0};
-  if(information.board == "MSX#A") board = new Board::MSX{*this, 1};
+  if(information.board == "Nemesis") board = new Board::MSX{*this, 1}; // Nemesis (Korea)
   if(information.board == "Janggun") board = new Board::Janggun{*this};
+  if(information.board == "Hicom") board = new Board::Hicom{*this};
+  if(information.board == "pak4") board = new Board::Pak4{*this}; // 4 PAK All Action (Australia)
+  
+  // WIP
+  if(information.board == "Korea_NB") board = new Board::Korea{*this}; // TODO: Some of these games work but should not have bank switching.
```

Comment:
consider making board::korea take additional parameters, to specify weather banking should be enabled and/or how many banks to allocate

This information can come from the game database or manifest, see how the MSX board accepts 0 or 1 as a parameter to determine which features to use? you could do the same here.